alacritty_terminal/src/grid/storage.rs

```diff
@@ -62,12 +62,12 @@ impl<T: PartialEq> PartialEq for Storage<T> {
 
 impl<T> Storage<T> {
     #[inline]
-    pub fn with_capacity(visible_lines: Line, template: Row<T>) -> Storage<T>
+    pub fn with_capacity(visible_lines: Line, cols: Column) -> Storage<T>
     where
-        T: Clone,
+        T: GridCell + Default + Clone,
     {
-        // Initialize visible lines, the scrollback buffer is initialized dynamically.
-        let inner = vec![template; visible_lines.0];
+        // Initialize visible lines; the scrollback buffer is initialized dynamically.
```

Comment:
It's correct.

Reply:
Yeah, it's basically a completely separate sentence but it is still related to the first one, that's why I've put the semicolon there. With a comma I'd have to put a `since` or something in between for it to read nicely imo.